monai/apps/detection/transforms/array.py

```diff
@@ -373,3 +382,72 @@ def __call__(  # type: ignore
             labels_t = deepcopy(labels_t[keep_t, ...])
             labels_clip_list.append(convert_to_dst_type(src=labels_t, dst=labels_tuple[i])[0])
         return boxes_clip, tuple(labels_clip_list)
+
+
+class BoxToBoxMask(Transform):
+    """
+    Convert box to int16 mask image, which has the same size with the input image.
+
+    Args:
+        bg_label: background labels for the output mask image, make sure it is smaller than any foreground(fg) labels.
+        ellipse_mask: bool.
+            If True, it assumes the object shape is close to ellipse or ellipsoid.
+            If False, it assumes the object shape is close to rectangle or cube and well occupies the bounding box.
+            If the users are going to apply random rotation as data augmentation, we suggest setting ellipse_mask=True
+            See also Kalra et al. "Towards Rotation Invariance in Object Detection", ICCV 2021.
+    """
+
+    backend = [TransformBackends.NUMPY]
+
+    def __init__(self, bg_label: int = -1, ellipse_mask: bool = False) -> None:
+        self.bg_label = bg_label
+        self.ellipse_mask = ellipse_mask
+
+    def __call__(  # type: ignore
+        self, boxes: NdarrayOrTensor, labels: NdarrayOrTensor, spatial_size: Union[Sequence[int], int]
+    ) -> NdarrayOrTensor:
+        """
+        Args:
+            boxes: bounding boxes, Nx4 or Nx6 torch tensor or ndarray. The box mode is assumed to be ``StandardMode``.
+            labels: classification foreground(fg) labels corresponding to `boxes`, dtype should be int, sized (N,).
+            spatial_size: image spatial size.
+
+        Return:
+            - int16 array, sized (num_box, H, W). Each channel represents a box.
+                The foreground region in channel c has intensity of labels[c].
+                The background intensity is bg_label.
+        """
+        return convert_boxes_to_masks(boxes, labels, spatial_size, self.bg_label, self.ellipse_mask)
```

Comment:
I feel it's confusing that the class name is "box" and "mask", but the actual function is "boxes" and "masks".
Could you please help unify them? Maybe change to the function name to: `convert_box_to_mask()`?
Same comments for all the other similar cases in the detection module.

Thanks.